sequelize/seeders/20180711143600-memeolist-example.js

```diff
@@ -17,20 +17,30 @@ const notesSchema = {
   name: 'default',
   schema: `
   
-  schema {
-    query: Query
-    mutation: Mutation
-    subscription: Subscription
+  type Profile {
+    id: ID! @isUnique
```

Comment:
Usually generated code contains fragement. Pattern is that frament is being used to defined type. Then it's also referenced in every crud operation. Schema defined this way is easier to maintain